src/backend/distributed/executor/multi_router_executor.c

```diff
@@ -534,6 +536,7 @@ RouterSelectExecScan(CustomScanState *node)
 {
 	CitusScanState *scanState = (CitusScanState *) node;
 	TupleTableSlot *resultSlot = NULL;
+	uint64 totalIntermediateResultSize = 0;
```

Comment:
Would it make sense to keep this counter in `ExecuteSingleSelectTask`? It seems to mimic `currentAffectedTupleCount`.